package.json

```diff
@@ -10,7 +10,7 @@
     "confit": "^2.0.0",
     "debug": "^2.1.0",
     "lodash": "^4.12.0",
-    "selenium-webdriver": "^2.53.2",
+    "selenium-webdriver": "^3.0.0-beta-2",
```

Comment:
Hi @gkushang can you modify this to `^3.0.0` since selenium 3 is out of beta now?